server/lemmy_utils/src/settings.rs

```diff
@@ -133,6 +133,19 @@ impl Settings {
     fs::read_to_string(CONFIG_FILE)
   }
 
+  pub fn get_allowed_instances(&self) -> Vec<String> {
+    let mut allowed_instances: Vec<String> = self
+      .federation
+      .allowed_instances
+      .split(',')
+      .map(|d| d.to_string())
+      .collect();
+
+    allowed_instances.retain(|d| !d.eq(""));
```

Comment:
I noticed the defaults.hjson config always returns a [""], (the list is a comma string) so we need to remove that one.

Reply:
Better put this kind of note in a comment tbh.

Reply:
Done